config/environments/development.rb

```diff
@@ -15,6 +15,37 @@
   # Log error messages when you accidentally call methods on nil.
   config.whiny_nils = true
 
+  config.log_level = :debug
+
+  # Basic log config, for calls to Rails.logger.<level> { <message> }
+  config.logger = ::Logger.new(STDOUT)
+  config.logger.formatter = ->(severity, datetime, progname, msg) { "#{severity[0]} #{msg}\n" }
```

Comment:
Shouldn't be worries with this, as the logger is always invoked with `log.<debug|info|warn|error|fatal>` which add's the severity.